### PR TITLE
Fix background of thinkpad/lenovo bbs

### PR DESCRIPTION
**Pull request by felixonmars**
There is a `!important` style to set background on `<body>` tag, so I managed to workaround it in an ugly way.

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -561,6 +561,16 @@ INVERT
 
 ================================
 
+bbs.thinkpad.com
+club.lenovo.com.cn
+
+CSS
+body[style] {
+    box-shadow: inset 0px 0px 0px 9999px var(--darkreader-neutral-background); }
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:569]: `}` remove this.

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -561,6 +561,16 @@ INVERT
 
 ================================
 
+bbs.thinkpad.com
+club.lenovo.com.cn
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:565]: `.cn` Not needed.

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (felixonmars) [reply src/config/dynamic-theme-fixes.config]: Oops, overlooked this. Removed.

File: src/config/dynamic-theme-fixes.config (hunk 2 above)
Comment (felixonmars) [reply src/config/dynamic-theme-fixes.config:565]: It's actually needed in https://club.lenovo.com.cn/think or the individual forum post page like https://club.lenovo.com.cn/thread-5954537-1-1.html

**Pull request merged**